### PR TITLE
Button element

### PR DESCRIPTION
**Pull request by ritik-patel05**
https://user-images.githubusercontent.com/67374176/151669517-4905b9bd-b233-45d0-8890-84ad8c9cac98.mp4

### CODE REVIEW

File: css/elements/button.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    
+    /* inline so that, it remains in line. */
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+
+    padding: var(--size-4) var(--size-9);
+
+    border: var(--size-1) solid transparent;
```
Comment (Hamdrive) [inline css/elements/button.css]: Having transparent border makes it redundant?

File: css/elements/button.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+/* large */
+.is-lg {
+    font-size: var(--size-11);
+}
+
+/* ---- Full screen display ---- */
+
+/* takes complete width of parent */
+.btn.is-wholewidth {
```
Comment (Hamdrive) [inline css/elements/button.css]: Use `vw` instead of wholewidth, specifies the same thing

**Review by Hamdrive — COMMENTED** [review]
Great use of comments to understand CSS styling used

**Review by shivampandey0 — COMMENTED** [review]
Looks all good to me.

**Review by junaidshaikh-js — COMMENTED** [review]
Everything looks good!

**Pull request merged**